federatedrec/README.md

```diff
@@ -0,0 +1,30 @@
+### Federated Recommondation
+
+#### 1. Introduction
+
+Federatedrec aims to address recommondation problems such as rating prediction and item ranking under federated learning senario. It includes implementation of a number of popular recommondation algorithms based on FATE library. Such as federated fm, federated mf, federated svd etc.
+
+#### 2. Background
```

Comment:
"Background" session is not relevant to the module function. Please consider remove this session from the document.